src/sentry/static/sentry/app/views/organizationIntegrations/integrationRow.tsx

```diff
@@ -173,4 +188,19 @@ const CategoryTag = styled(
   color: ${p => (p.priority ? p.theme.white : p.theme.gray4)};
 `;
 
+const ResolveNowButton = styled(Button)`
+  color: ${p => p.theme.gray2};
+  background: #ffffff;
+
+  border: ${p => `1px solid ${p.theme.gray2}`};
+  box-sizing: border-box;
+  box-shadow: 0px 2px 1px rgba(0, 0, 0, 0.08);
+  border-radius: 4px;
+  float: right;
```

Comment:
😬 float scare me, can we use grid on the alert container?

Reply:
No, couldn't figure out how :)